packages/api/src/command/medical/facility/create-facility.ts

```diff
@@ -34,24 +36,15 @@ export const createFacility = async ({
 };
 
 export function validateCreate(facility: FacilityCreate, throwOnError = true): boolean {
```

Comment:
this is only on create - should we have this on update as well?

Reply:
We do have it on the [validateUpdate](https://vscode.dev/github/metriport/metriport/blob/1785-ci-query-pattern-patch/packages/api/src/command/medical/facility/update-facility.ts#L63-L64)

Reply:
I mean all of the validation that will throw `BadRequestError` if the combination of values aren't valid